layout/sass/theme_frogg.sass

```diff
@@ -22,10 +22,10 @@ $footcolor: #eee
 $nontdcolor: #EEDDCF
 $percfaded: #777
 
-$barbg: #549C71
-$baraltbg: #549C71
+$barbg: #161
```

Comment:
please don't reduce hex colors to 3 digits unless they're all the same digit (FFF->fff is okay; this is not)

readability counts :)